core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -49,33 +54,21 @@
   private static final String LOST_WORKER_STATE = "Out of Service";
   private static final int BLOCK_SIZE_LIMIT = 100;
 
-  /** Worker's address. */
-  private final WorkerNetAddress mWorkerAddress;
-  /** The id of the worker. */
-  private final long mId;
-  /** Start time of the worker in ms. */
-  private final long mStartTimeMs;
-  /** Capacity of worker in bytes. */
-  private long mCapacityBytes;
-  /** Worker's used bytes. */
-  private long mUsedBytes;
-  /** Worker's last updated time in ms. */
-  private long mLastUpdatedTimeMs;
-  /** If true, the worker is considered registered. */
-  private boolean mIsRegistered;
-  /** Worker-specific mapping between storage tier alias and storage tier ordinal. */
-  private StorageTierAssoc mStorageTierAssoc;
-  /** Mapping from storage tier alias to total bytes. */
-  private Map<String, Long> mTotalBytesOnTiers;
-  /** Mapping from storage tier alias to used bytes. */
-  private Map<String, Long> mUsedBytesOnTiers;
+  /** worker metadata */
+  private final WorkerMeta mMeta;
```

Comment:
`GuardedBy("mMetaLock")` annotation?
Also, not sure if you really need to protect `WorkerMeta` which has only one mutable info `mLastUpdatedTimeMs`.
In that case, you can move the `mLastUpdatedTimeMs` to `UsageMeta` (or rename it to something better)

Reply:
Yea I'm thinking about doing that too